pkg/helpers/helpers.go

```diff
@@ -152,6 +122,23 @@ func DecodeSnowflakeParameterID(identifier string) (sdk.ObjectIdentifier, error)
 	}
 }
 
+// DecodeSnowflakeAccountIdentifier decodes account identifier (usually passed as one of the parameter in tf configuration) into sdk.AccountIdentifier.
+// Check more in https://docs.snowflake.com/en/sql-reference/sql/create-account#required-parameters.
+func DecodeSnowflakeAccountIdentifier(identifier string) (sdk.AccountIdentifier, error) {
+	parts, err := ParseIdentifierString(identifier)
+	if err != nil {
+		return sdk.AccountIdentifier{}, err
+	}
+	switch len(parts) {
+	case 1:
+		return sdk.AccountIdentifier{}, fmt.Errorf("identifier: %s seems to be account locator and these are not allowed - please use <organization_name.account_name>", identifier)
+	case 2:
+		return sdk.NewAccountIdentifier(parts[0], parts[1]), nil
+	default:
+		return sdk.AccountIdentifier{}, fmt.Errorf("unable to classify account identifier: %s", identifier)
```

Comment:
maybe it's also worth here to mention the expected format ?

Reply:
oky